### ISSUE TITLE
Firefox for direct on Android tv unable to pin tile 

### ISSUE THREAD

**Issue by melmoxyz** [mozilla-mobile/firefox-tv#2977]
## Steps to reproduce
Pin tile icon is already grey.
### Expected behavior
Pressing menu should provide option to pin tile
### Actual behavior
Menu button brings up tv menu not Firefox menu.  
### Device information -Hisense AndroidTV side loaded api 4.6 for fire tv
* Fire TV device: ? No
* Latest affected Firefox version: ? 4.6
* Earliest affected Firefox version: ?
* Turbo Mode: ? On

**Comment by athomasmoz** [comment]
Hi @melmoxyz thanks for logging. Unfortunately, this is because we rely on the menu (3 line) icon on Fire TV remotes to get to the menu. Android TV is not officially supported, but something we may explore in future #2660 . Linking to this issue as a reminder that we would need to find a solution

**Comment by max1c** [comment]
@athomasmoz is there any kind of temporary workaround?

**Comment by rimas-kudelis** [comment]
@max1c perhaps an "tv remote" android apps will have the menu button?
I've looked over a few of them just now, but found nothing workable so far. I also tried a button remapper, but the only button on my remote that I wouldn't miss is the "speak" button, or whatever its name, and the remapper was unable to intercept it. Lucky me I have two different MiBox remotes, and one of them (the Chinese version) actually has the menu button, so I was finally able to pin something today. :)

**Comment by max1c** [comment]
> @max1c perhaps one of "TV Remote" apps for Android (or iOS) will have the menu button?
> I've looked over a few of them just now, but found nothing workable so far. I also tried a button remapper, but the only button on my remote that I wouldn't miss is the "speak" button, or whatever its name, and the remapper was unable to intercept it. Lucky me I have two different MiBox remotes, and one of them (the Chinese version) actually has the menu button, so I was finally able to pin something today. :)

Yes, Cetus app has the 3 line menu and it works as a workaround for Firefox. Cetus app is horrible though.

**Comment by rimas-kudelis** [comment]
Returning back to the original issue, I think the most obvious way to replace the menu button on Android TV is to show the menu on long click of the select button. At least that's how you trigger what I would call context menu on Android TV homescreen.
